validation-test/Sema/type_checker_perf/fast/rdar33292740.swift.gyb

```diff
@@ -1,4 +1,4 @@
-// RUN: %scale-test --invert-result --begin 2 --end 7 --step 1 --select incrementScopeCounter %s
+// RUN: %scale-test --begin 2 --end 10 --step 1 --select incrementScopeCounter %s
```

Comment:
It's a bit surprising that this would have a real effect on whether this is fast or slow. Are you confident this is a real improvement, or was this perhaps not really slow before (e.g. it was slow over that range but wasn't truly exponential?).

Reply:
It's a real improvement because this triggers logic in the `solveSimplified` where solver is not going to look at generic overloads if it could find non-generic one with the same score, since we are not increasing the score here some of the cases are going to behave better than before.

Reply:
Makes sense.